pynxtools/dataconverter/readers/xrd/README.md

```diff
@@ -0,0 +1,40 @@
+# XRD reader
```

Comment:
Why is this part of this PR?
Please remove the xrd part and make it separate PR if necessary.